### ISSUE TITLE
image not found when installed via SwiftPM

### ISSUE THREAD

**Issue by laosb** [GEOSwift/GEOSwiftMapKit#8]
> Split out from #4

Just tried adding GEOSwiftMapKit to my Xcode iOS project via Xcode SwiftPM integration, and it failed when I try to run:

```
dyld: Library not loaded: @rpath/libgeos.dylib
  Referenced from: /private/var/containers/Bundle/Application/[REDACTED]/[REDACTED].app/[REDACTED]
  Reason: image not found
(lldb) 
```

Versions:

![image](https://user-images.githubusercontent.com/2545261/69425936-e0f1f500-0d66-11ea-9351-3af03e5c4d74.png)

Those are figured out by Swift PM itself.

And it works charmingly on simulator, but only leaves such an error on application starting when tested on an actual device.

**Comment by macdrevx** [comment]
@laosb We've seen a [similar issue](https://github.com/GEOSwift/GEOSwift/issues/195) that was resolved by explicitly adding geos as a dependency. If that doesn't work for you, can you share a sample project that demonstrates this issue?

**Comment by macdrevx** [comment]
@laosb the other issue I mentioned seems to have been resolved by Xcode 11.4. Can you let me know if you're still experiencing this issue?

**Comment by macdrevx** [comment]
@laosb since I haven't heard back from you, I'm going to close this issue. Feel free to reopen it if you think this is still a problem.
